### PR TITLE
Remove setting obsolete App->data['user'] in Module\Profile\Contacts

### PR DESCRIPTION
**Pull request by MrPetovan**
Addresses https://github.com/friendica/friendica/issues/7298#issuecomment-533765982

### CODE REVIEW

File: src/Util/Temporal.php [hunk 1]
```diff
@@ -122,13 +122,12 @@ public static function getTimezoneField($name = 'timezone', $label = '', $curren
 	 * @brief Wrapper for date selector, tailored for use in birthday fields.
 	 *
 	 * @param string $dob Date of Birth
+	 * @param string $timezone
 	 * @return string Formatted HTML
-	 * @throws \Friendica\Network\HTTPException\InternalServerErrorException
+	 * @throws \Exception
 	 */
-	public static function getDateofBirthField($dob)
+	public static function getDateofBirthField($dob, $timezone = 'UTC')
```
Comment (nupplaphil) [inline src/Util/Temporal.php]: Typehint `string` and `string` :)

Comment (MrPetovan) [reply src/Util/Temporal.php]: Isn't it useless since it's the lowest constraint type? Like everything is string by default?

Comment (nupplaphil) [reply src/Util/Temporal.php]: What about `False` is given as argument? Would it create an exception of would it be casted to an empty string? Or would it be passed as boolean? I've the fear that the last thing would occur and than at another point of the code, a boolean type isn't valid and create an exception at other places.

Comment (MrPetovan) [reply src/Util/Temporal.php]: Fair enough, I thought it was the lowest common parameter type denominator and as such could be skipped.

**Pull request merged**